community/caddy/caddy.initd

```diff
@@ -3,11 +3,46 @@
 name="Caddy webserver"
 command="/usr/sbin/caddy"
 command_args="$CADDY_OPTS"
-pidfile="/var/run/caddy.pid"
 command_background=yes
-start_stop_daemon_args="--user caddy --group caddy"
+command_user=caddy:caddy
+extra_commands="checkconfig"
+extra_started_commands="reload"
+
+# Pidfile only required for start-stop-daemon.
+if [ -z "$supervisor" ]; then
+	pidfile="/var/run/caddy.pid"
+fi
 
 depend() {
 	need net localmount
 	after firewall
 }
+
+checkconfig() {
+	ebegin "Checking configuration for $RC_SVCNAME"
+	su -s /bin/sh ${command_user%:*} -c "$command $command_args -validate"
+	eend $?
+}
+
+reload() {
+	checkconfig || return $?
+	ebegin "Reloading $RC_SVCNAME"
+	case "$supervisor" in
+	"") start-stop-daemon --signal USR1 --pidfile "$pidfile" ;;
```

Comment:
Please indent `PATTERN)` sections, and do not leave a space before the `;;`s.
Ditto for the next two lines.